courses/machine_learning/deepdive2/feature_engineering/solutions/4_keras_adv_feat_eng.ipynb

```diff
@@ -0,0 +1,1751 @@
+{
```

Comment:
<p>I'd remove reference to the solution notebook.Since this is the solution notebook. </p><p> </p><p>Also, as in previous labs, the Learning Objectives here will correspond to TODOs or code for students to fill in. This seems like a lot. We may want to limit to just the parts that correspond to feature engineering in keras. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Removed reference to the solution notebook.  Also, learning objectives are now:</p><ol><li>Create an input pipeline using tf.data</li><li>Engineer features to create categorical, numerical, temporal, and geolocation features</li></ol><p> </p><p> </p>